src/themes/luminara.css

```diff
@@ -72,7 +72,10 @@ body[data-theme="luminara"] {
 
   .product-detail-image {
     border-radius: 16px;
-    max-width: 300px;
+
+    @media screen and (min-width: 576px) {
```

Comment:
Not sure why we want this on only 2 themes as it's mentioned the image should be full width.
<img width="516" alt="image" src="https://github.com/user-attachments/assets/aaea6024-e78f-4003-ad7f-ad6510f2af6e">

Full width
<img width="482" alt="image" src="https://github.com/user-attachments/assets/c1a9e2b2-497f-4187-8c83-f8f020c2c4e2">

Reply:
@WiNloSt The idea from discussing with Alessio is that we should only make it full-width on the mobile (base) viewport, therefore `xs` and above should start to constrain the image's width. Otherwise it looks way too big on screens larger than xs (576px).

So, the fix should actually be to also make the `product-detail-image` CSS global (as opposed to being specific to theme), so it also applies to theStitch (the dark theme)

Reply:
OK, so I misread that style, still the issue is that themes other than the first site don't have full-width images on product detail page which should be fixed.

Reply:
@WiNloSt not sure what viewport sizes you are using, but the design requirement is to use full-width images **only on viewport below 576px**, which it currently does.

Viewport of 544px:

![CleanShot 2567-10-07 at 20 07 35@2x](https://github.com/user-attachments/assets/68b3dc9f-a756-46a7-a1dd-260e4db08bef)

Viewport of 642px:

![CleanShot 2567-10-07 at 20 07 39@2x](https://github.com/user-attachments/assets/ce9965da-fd6a-448e-8cb3-9c05359ef7a4)

The above is expected. So, it's correct on Luminar and Pug & Play, but not on theStitch (dark theme)

Reply:
> @WiNloSt not sure what viewport sizes you are using, but the design requirement is to use full-width images **only on viewport below 576px**, which it currently does.
> 
> Viewport of 559px:
> 
> ![CleanShot 2567-10-07 at 20 07 35@2x](https://private-user-images.githubusercontent.com/4714175/374180811-68b3dc9f-a756-46a7-a1dd-260e4db08bef.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjgzMDY5MjAsIm5iZiI6MTcyODMwNjYyMCwicGF0aCI6Ii80NzE0MTc1LzM3NDE4MDgxMS02OGIzZGM5Zi1hNzU2LTQ2YTctYTFkZC0yNjBlNGRiMDhiZWYucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTAwNyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDEwMDdUMTMxMDIwWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NGY0NjEwNzI1OGMxYjI1YjQwYzJiNzJkYjQwMTRlOTViMTI4ZjE1MzgyMGNhNTJkMmQzNjRhODViOTgwMmU1ZiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.ZZcSuVwcQzjUVuJgLJthSrmF0iFUKgLPa56qt0fgHn0)
> 
> Viewport of 681px:
> 
> ![CleanShot 2567-10-07 at 20 07 39@2x](https://private-user-images.githubusercontent.com/4714175/374180796-ce9965da-fd6a-448e-8cb3-9c05359ef7a4.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjgzMDY5MjAsIm5iZiI6MTcyODMwNjYyMCwicGF0aCI6Ii80NzE0MTc1LzM3NDE4MDc5Ni1jZTk5NjVkYS1mZDZhLTQ0OGUtOGNiMy05YzA1MzU5ZWY3YTQucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTAwNyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDEwMDdUMTMxMDIwWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZGMwYjZkODIzMzJlNTM0ZTg1YzE2ZDhiNzI3MDU1NzhhNmM1M2UxYWJjN2IzZjk3MTZmZDFiN2Q3ODlmNWEwZiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.PY9ZvvRfjXyoWof-deOROwFJ0DRN-41Y5uDfxBIU8FA)
> 
> The above is expected. So, it's correct on Luminar and Pug & Play, but not on theStitch (dark theme)

That explains it. Thanks! I think the viewport I use might be larger than that, so the I mistook the full-width on Stitch theme as the correct one, when it's the one that needs to be fixed.

Reply:
Fixed in https://github.com/metabase/shoppy/pull/68/commits/c34eb0cec20ce88c3c4fde2cbe0d09f62499ef95